lib/logcat.js

```diff
@@ -56,6 +61,8 @@ class Logcat {
         }
       });
       await this.proc.start(0);
+      // resolve after a timeout, even if no output was recorded
+      setTimeout(resolve, LOGCAT_PROC_STARTUP_TIMEOUT);
```

Comment:
is it ok to assume the promise is resolved after the timeout? I'd rather say it should be rejected

Reply:
It's not really an error condition. If there is an error the correct event will be fired, and we'll reject. This is just the case where the device does not produce any logs after we start listening for them. This has been happening in Travis in particular.